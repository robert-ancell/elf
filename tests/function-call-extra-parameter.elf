uint8 foo () {
   return 42
}
foo (123)
