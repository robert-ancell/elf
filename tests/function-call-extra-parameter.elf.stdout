Line 4:
foo (123)
    ^
Function takes no parameters
